### DISCUSSION TITLE
range component

### DISCUSSION THREAD

**Discussion by wibed** [saadeghi/daisyui#692] [Q&A] [answered]
the range component wont update its length moving, even though the state is updated and the component refreshed.
ill dig into it, but just posting it out there maybe someone has already done this and can explain.

**Comment by saadeghi** [comment, marked as answer]
Can you provide an example repo so I can reproduce the issue?

**Reply by wibed** [reply]
nope, my bad i did not correctly implement the component. first task in the morning :P
Love your work btw
